.github/workflows/ci.yml

```diff
@@ -15,48 +15,66 @@ jobs:
       - name: Setup Deno environment
         uses: denolib/setup-deno@v1.3.0
         with:
-          deno-version: v1.2.2
+          deno-version: v1.3.0
```

Comment:
not 1.3.2?

Reply:
I tested with 1.3.0, will update to 1.3.2 later, once I tested that.